integration-tests/parser/indentation/expr-needing-indentation/src/Main.elm

```diff
@@ -0,0 +1,6 @@
+module Main exposing (MyMaybe)
+
+
+main =
+  1
++ 7
```

Comment:
:+1: I think we can make this test better!

It tests the "trivial" case where indentation is 0; I'd like to test it (perhaps in a separate test?) where the indentation is different from 0. Eg. something like:`let..in` which sets the indentation to the binding level (where `x` is).

FAIL :x:
```elm
main =
    let
        x =
            1
        + 2
    in
    x
```

PASS :heavy_check_mark: 
```elm
main =
    let
        x =
            1
         + 2
    in
    x
```
(Tested on official compiler in Ellie)

Reply:
The problem is that for failing tests we can only have one failure per file so testing multiple failure modes quickly gets a bit much

Reply:
We should for example test other operators such as `++`